Cargo.toml

```diff
@@ -43,14 +43,18 @@ informalsystems-pbjson = { version = "0.7.0", optional = true, default-features
 version          = "0.39"
```

Comment:
Should we remove ics23 dep? I don't see it being exported anymore.

Reply:
It's now re-exported again, given that we need to depend on it anyhow